.github/workflows/tutorials.yml

```diff
@@ -0,0 +1,45 @@
+name: Tutorials
+
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - "examples/tutorials/**"
+    - "!examples/tutorials/translations/**"
+
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+    - "examples/tutorials/**"
+    - "!examples/tutorials/translations/**"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        python -m pip install --upgrade pip
+        hash -r
+        pip3 install -r pip-dep/requirements.txt
+        pip3 install -r pip-dep/requirements_dev.txt
+        pip3 install -r pip-dep/requirements_udacity.txt
+        pip3 install -r pip-dep/requirements_notebooks.txt
+        pip3 install scikit-learn
+        python setup.py install
+    - name: Run the tutorials
+      run: |
+        pytest test/notebooks/test_notebooks.py -m "not translation"
```

Comment:
`-m "not translation"` excludes all the tests that are marked as `translation` tests, so this runs all the notebook tests except the translation tests (which have their own action.)